data/migrations/01_problems.js

```diff
@@ -0,0 +1,30 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('problems', (tbl) => {
+    tbl.increments();
+    tbl
+      .string('problem_title')
+      .notNullable();
+    tbl
+      .string('problem_description')
+      .notNullable();
+    tbl
+      .string('problem_category') // if dropdown is it still a string?
+      .notNullable();
+    tbl
+      .integer('date') // for dates is int correct?
```

Comment:
I'll update this to .date and still looking into the dropdown category.

Reply:
👍